Gemfile.lock

```diff
@@ -1,34 +0,0 @@
-PATH
```

Comment:
You should not include `Gemfile.lock` within the Gem. 

This is the [first link](https://depfu.com/blog/2017/01/18/bundler-and-gemfile-best-practices#gemfilelock-is-for-apps-not-libraries) i've found on Google. But you can find more about it.